CI run failed at [`Unhandled error: Error: Cannot find module '@actions/github'`](https://github.com/marilyn79218/react-lazy-show/pull/44/checks?check_run_id=2978437648), closing PR and try to use `pull_request_target` event for dependabot 
 to write comments.

--- a/.github/workflows/auth-writing-comments.js
+++ b/.github/workflows/auth-writing-comments.js
@@ -1,0 +1,50 @@
+// It's a testing script to test dependabot writing permission
+// Should be removed once the test done.
+
+const COMMENT_ANCHOR = "dependabot_test";
+
+module.exports = async (github, context, core, octokit) => {
+  try {
+    console.log("octokit", octokit);
+    console.log("context", context);
+
+    // Find comment id if exist
+    const { data: existingComments } = await octokit.issues.listComments({
+      issue_number: context.issue.number,
+      owner: context.repo.owner,
+      repo: context.repo.repo,
+    });
+
+    let commentId;
+    for (let i = existingComments.length; i--; ) {
+      const c = existingComments[i];
+      if (c.user.type === "Bot" && c.body.includes(COMMENT_ANCHOR)) {
+        commentId = c.id;
+        break;
+      }
+    }
+
+    const date = new Date();
+    const localTimeString = date.toLocaleString();
+    const commentBody = `${COMMENT_ANCHOR}: ${context.actor} - ${context.sha}`;
+    if (!commentId) {
+      console.log("Creating comment...");
+      await octokit.issues.createComment({
+        issue_number: context.issue.number,
+        owner: context.repo.owner,
+        repo: context.repo.repo,
+        body: commentBody,
+      });
+    } else {
+      console.log("Updating comment...", commentId);
+      await octokit.issues.updateComment({
+        comment_id: commentId,
+        owner: context.repo.owner,
+        repo: context.repo.repo,
+        body: commentBody,
+      });
+    }
+  } catch (error) {
+    core.setFailed(error.message);
+  }
+}s

--- a/.github/workflows/pr-auth-dependabot.yml
+++ b/.github/workflows/pr-auth-dependabot.yml
@@ -22,5 +22,5 @@ jobs:
             const path = require('path');
             const scriptPath = path.resolve('./.github/workflows/writing-comments.js');
             const { getOctokit } = require(`${process.env.GITHUB_WORKSPACE}/node_modules/@actions/github`)
-            const octokit = getOctokit(${{ secrets.GITHUB_TOKEN }});
+            const octokit = getOctokit('${{ secrets.GITHUB_TOKEN }}');
             await require(scriptPath)(github, context, core, octokit);

--- a/.github/workflows/pr-auth-dependabot.yml
+++ b/.github/workflows/pr-auth-dependabot.yml
@@ -1,0 +1,25 @@
+name: auth dependabot myself
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  dependabot-comments:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "14.15.4"
+      - name: "Writing comments"
+        uses: actions/github-script@v4
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const path = require('path');
+            const scriptPath = path.resolve('./.github/workflows/writing-comments.js');
+            const { getOctokit } = require(`${process.env.GITHUB_WORKSPACE}/node_modules/@actions/github`)
+            const octokit = getOctokit(${{ secrets.GITHUB_TOKEN }});
+            await require(scriptPath)(github, context, core, octokit);

--- a/.github/workflows/pr-auth-dependabot.yml
+++ b/.github/workflows/pr-auth-dependabot.yml
@@ -13,6 +13,7 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: "14.15.4"
+      - run: yarn install
       - name: "Writing comments"
         uses: actions/github-script@v4
         with:

--- a/package.json
+++ b/package.json
@@ -33,6 +33,7 @@
     ]
   },
   "devDependencies": {
+    "@actions/github": "^5.0.0",
     "babel-cli": "^6.26.0",
     "babel-core": "^6.26.0",
     "babel-eslint": "^8.2.1",

